tests/app/filter.test.ts

```diff
@@ -91,15 +91,15 @@ for (const edgeCase of TESTS) {
   test(`${edgeCase.desc}`, async ({ page }) => {
     await loadMap(page);
 
-    // Deal with the requirements toggle before the filter popup, since the
-    // filter popup covers it.
+    // Default has requirement toggle on, so first de-select it, but opening filter pop-up and clicking toggle.
```

Comment:
Do you mean `by opening` rather than `but opening`? Also the comment and code is repeated in some places - maybe deduplicate it as a helper function in `utils.ts`?